easybuild/easyconfigs/q/Qhull/Qhull-2015.2-intel-2016a.eb

```diff
@@ -0,0 +1,38 @@
+easyblock = 'CMakeMake'
+
+name = 'Qhull'
+version = '2015.2'
+
+homepage = 'http://www.qhull.org'
+description = """
+Qhull computes the convex hull, Delaunay triangulation, Voronoi diagram, halfspace intersection about a point,
+furthest-site Delaunay triangulation, and furthest-site Voronoi diagram. The source code runs in 2-d, 3-d, 4-d,
+and higher dimensions. Qhull implements the Quickhull algorithm for computing the convex hull.
+"""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+
+# Recently changed the Unix version numbering see http://www.qhull.org/src/Changes.txt
+# and https://github.com/Homebrew/homebrew-science/issues/3353
+sources = ['%(namelower)s-2015-src-7.2.0.tgz']
```

Comment:
it seems that:
2015.2 = 7.2.0
2015.1 = 7.1.0
2015.0.7 = 7.0.7
2015.0.6 = 7.0.6.2013
2015.0.5 = 7.0.5.1995
2015.0.4 = 7.0.4.1984
2015.0.3
2015.0.2
2015.0.1
2012.1  6.3.1.1494 (2012/02/18)
2012.1  6.3.1.1490 (2012/02/18)
2012.1  6.3.0.1483 (2012/02/02)
etc.

so maybe we have to refer as `qhull_VERSION`, and this is not consistent at all.